netmaster/master/network.go

```diff
@@ -124,6 +125,14 @@ func createDockNet(tenantName, networkName, serviceName, subnetCIDR, gateway str
 		return errors.New("Network name used by another driver")
 	}
 
+	netPluginOptions := make(map[string]string)
+	netPluginOptions["tenant"] = nwCfg.Tenant
+	netPluginOptions["encap"] = nwCfg.PktTagType
+	netPluginOptions["pkt-tag"] = strconv.Itoa(nwCfg.PktTag)
+	if nwCfg.PktTagType == "vxlan" {
```

Comment:
We should display only the options that are specified by the user. i.e. the tag user specified we should include, otherwise internal stuff skip it. especially extPktTag.